tests/test_board.py

```diff
@@ -1,60 +1,142 @@
 import unittest
+
 from rookpoly import Board
-from rookpoly import Polynomial
+from rookpoly.polynomial import Polynomial
 
 
 class TestBoard(unittest.TestCase):
-
     empty_boards_limit = 7
 
     def setUp(self):
         self.empty_boards = [Board(['0' * i for _ in range(i)]) for i in range(self.empty_boards_limit)]
-        self.board_1 = Board(["110", "100", "000", "001"])
-        self.board_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_1 = Board(["110", "100", "000", "001"])
+        self.board_global_2 = Board(["00111", "00111", "11100", "11100", "11000"])
+
+    def test_init(self):
+        # Testowanie poprawnej inicjalizacji planszy
+        board = Board(["100", "010"])
+        self.assertEqual(board.n, 2)
+        self.assertEqual(board.m, 3)
+
+    def test_n(self):
+        # Testowanie właściwości n, zwracającej liczbę wierszy
+        self.assertEqual(self.board_global_1.n, 4)
+        self.assertEqual(self.board_global_2.n, 5)
+
+    def test_m(self):
+        # Testowanie właściwości m, zwracającej liczbę kolumn
+        self.assertEqual(self.board_global_1.m, 3)
+        self.assertEqual(self.board_global_2.m, 5)
+
+    def test_flip(self):
+        # Testowanie odwracania planszy
+        board = Board(["100", "010", "001"])
+        board.flip()
+        self.assertEqual(board, Board(["001", "010", "100"]))
 
     def test_rotate(self):
+        # Testowanie rotacji planszy
         for (i, board) in enumerate(self.empty_boards):
             board.rotate()
             self.assertEqual(board, ['0' * i for _ in range(i)])
 
-        self.board_1.rotate()
-        self.assertEqual(self.board_1, ["0011", "0001", "1000"])
+        self.board_global_1.rotate()
+        self.assertEqual(self.board_global_1, ["0011", "0001", "1000"])
+
+        self.board_global_2.rotate()
+        self.assertEqual(self.board_global_2, ["11100", "11100", "01111", "00011", "00011"])
 
     def test_rtrim(self):
+        # Testowanie przycinania pustych miejsc po prawej stronie planszy
         for (i, board) in enumerate(self.empty_boards):
             board.rtrim()
             self.assertEqual(board, ['0' * i for _ in range(i)])
 
-        self.board_1.rtrim()
-        self.assertEqual(self.board_1, ["110", "100", "000", "001"])
+        self.board_global_1.rtrim()
+        self.assertEqual(self.board_global_1, ["110", "100", "000", "001"])
 
-        self.board_2.rtrim()
-        self.assertEqual(self.board_2, ["00111", "00111", "11100", "11100", "11000"])
+        self.board_global_2.rtrim()
+        self.assertEqual(self.board_global_2, ["00111", "00111", "11100", "11100", "11000"])
+
+    def test_remove_filled_rows(self):
+        # Testowanie usuwania wypełnionych wierszy z planszy
+        board = Board(["111", "000", "010"])
+        board.remove_filled_rows()
+        self.assertEqual(board, Board(["000", "010"]))
 
     def test_simplify(self):
+        # Testowanie uproszczenia planszy
```

Comment:
Maybe we should not add comments when the function name is self-explanatory

Reply:
I think we need to take the same approach to all tests